indico/modules/events/papers/operations.py

```diff
@@ -28,3 +31,22 @@ def set_reviewing_state(event, reviewing_type, enable):
     logger.info("Reviewing type '%s' for event %r %s by %r", reviewing_type, event, action, session.user)
     event.log(EventLogRealm.management, EventLogKind.positive, 'Papers',
               "{} reviewing type '{}'".format("Enabled" if enable else "Disabled", reviewing_type), session.user)
+
+
+def create_competences(event, user_id, competences):
+    user_competences = PaperCompetence(user_id=user_id, event_id=event.id, competences=competences)
+    db.session.add(user_competences)
+    user = User.get_one(user_id)
```

Comment:
I think this should be outside, and the function should take a `user`. That way you can also pass the user to the `PaperCompetence` constructor. Passing IDs around for relationships is a bad idea in general.